lib/app/features/home/match_details/match_details_module.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter_modular/flutter_modular.dart';
+import 'package:score_live/app/features/home/match_details/cubit/match_details_cubit.dart';
+import 'package:score_live/app/features/home/match_details/match_details_screen.dart';
+import 'package:score_live/data/api.client.dart';
+import 'package:score_live/data/live_matches_remote_service.dart';
+import 'package:score_live/repositories/match_details_repository.dart';
+
+class MatchDetailsModule extends Module {
+  @override
+  List<Bind> get binds => [
+        Bind.factory((i) => ApiClient()),
```

Comment:
Things like ApiClient, Services and Repositories should be singletons. And usually we keep those in separate modules which are linked to the main module in order to make sure those classes are available in the whole app. 

You can create f.e RepositoryModule and linked the module to the MainModule using imports propery like that:
  @override
  final List<Module> imports = [
    DIDataModule(),
    DINetworkModule(),
    DIBlocModule(),
    DIRepositoriesModule(),
    DIServicesModule(),
  ];